modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/model/ReferringSocialMedia.java

```diff
@@ -61,6 +61,15 @@ public JSONObject toJSONObject(ResourceBundle resourceBundle) {
 		);
 	}
 
+	@Override
```

Comment:
Hey @cgoncas, here we don't have title, as in the mocks (https://github.com/liferay-tango/liferay-portal/pull/674), is there any reason? 🤔

Reply:
Hey @dgarciasarai !!!

The toString method is not related whit the info you will receive in the frontend.

Thanks!!